packages/http-server/src/server.ts

```diff
@@ -36,11 +39,17 @@ export const createServer = (operations: IHttpOperation[], opts: IPrismHttpServe
     return done(error);
   });
 
-  const mergedConfig = configMergerFactory({ mock: { dynamic: false } }, config, getHttpConfigFromRequest);
+  const mergedConfig = configMergerFactory({ cors: false, mock: { dynamic: false } }, config, getHttpConfigFromRequest);
 
   const prism = createInstance(mergedConfig, components);
 
-  server.all('*', {}, replyHandler(prism));
+  opts.config.cors
+    ? server.route({
+        url: '*',
+        method: ['GET', 'DELETE', 'HEAD', 'PATCH', 'POST', 'PUT'],
```

Comment:
if we are going to whitelist methods we should at least have all the ones OpenAPI supports. 

Should we switch to `method: '*'` instead?

Reply:
Unfortunately we cannot use `*` because otherwise the CORS plugin would complain (you need to leave the `OPTIONS` verb free); so I put all the other verbs that Fastify supports. Any other will make the framework scream at me.